models/cart-discount/src/transformers.ts

```diff
@@ -0,0 +1,42 @@
+import { LocalizedString } from '@commercetools-test-data/commons';
+import { Transformer } from '@commercetools-test-data/core';
+import type { TCartDiscount, TCartDiscountGraphql } from './types';
+
+const transformers = {
+  default: Transformer<TCartDiscount, TCartDiscount>('default', {
+    buildFields: [
+      'value',
+      'name',
+      'description',
+      'createdBy',
+      'lastModifiedBy',
+    ],
```

Comment:
Same question here regarding fields we aren't technically using yet but could in the future (`target`, `references`, `custom`)

Reply:
Interesting, perhaps we could include these when we do eventually use models here? Right now there's nothing to build.

However, I should add these fields to the tests - just seems like good form to do so: see a919301024a2970cddb17ecd94dff85156e94c7d

Reply:
Works for me